interfaces/builtin/basedeclaration.go

```diff
@@ -326,6 +326,11 @@ slots:
       slot-snap-type:
         - core
     deny-auto-connection: true
+  media-hub:
+    allow-installation:
+      slot-snap-type:
+        - app
+    deny-auto-connection: true
```

Comment:
We allowed using media-hub on Touch and I expected us to allow this interface to be auto-connected. Why did you specify `deny-auto-connection: true` here?